### PR TITLE
New package: haxor-news-0.4.2

### PR DESCRIPTION
**Pull request by Crestwave**

### CODE REVIEW

File: srcpkgs/haxor-news/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'haxor-news'
+pkgname=haxor-news
+version=0.4.2
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="haxor_news"
+hostmakedepends="python3-setuptools"
+depends="python python3-click python3-colorama python3-requests python3-Pygments python3-prompt_toolkit python3-six"
```
Comment (maxice8) [inline srcpkgs/haxor-news/template]: remove python, cut at 80 columns

File: srcpkgs/haxor-news/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+short_desc="Browse Hacker News like a haxor: A Hacker News command line interface"
+maintainer="Crestwave <crest.wave@yahoo.com>"
+license="Apache-2.0"
+homepage="https://github.com/donnemartin/haxor-news"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum="fa35cea905c34b6975527b2e6e9d6b020a6cad6c664c8483128e10a07d998314"
```
Comment (maxice8) [inline srcpkgs/haxor-news/template]: no need to be quoted

File: srcpkgs/haxor-news/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'haxor-news'
+pkgname=haxor-news
+version=0.4.2
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="haxor_news"
+hostmakedepends="python3-setuptools"
+depends="python python3-click python3-colorama python3-requests python3-Pygments python3-prompt_toolkit python3-six"
+short_desc="Browse Hacker News like a haxor: A Hacker News command line interface"
```
Comment (maxice8) [inline srcpkgs/haxor-news/template]: maybe just 'Hacker News command line interface' ?

File: srcpkgs/haxor-news/template (hunk 2 above)
Comment (Crestwave) [reply srcpkgs/haxor-news/template]: Ah, the examples in the manual had them quoted so I thought that they were preferred. Thanks for the review!

File: srcpkgs/haxor-news/template [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'haxor-news'
+pkgname=haxor-news
+version=0.4.2
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="haxor_news"
```
Comment (cr6git) [inline srcpkgs/haxor-news/template]: `pycompile_module="haxor_news tests"`

Comment (asergi) [reply srcpkgs/haxor-news/template]: `tests` are outside the namespace and must be removed.

File: srcpkgs/haxor-news/template [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+hostmakedepends="python3-setuptools"
+depends="python3-click python3-colorama python3-requests python3-Pygments python3-prompt_toolkit python3-six"
```
[the remark this answers is not in the document]
Comment (Crestwave) [reply srcpkgs/haxor-news/template]: It does support Python 2.x.

**Pull request closed without merging**